src/main/java/uk/gov/hmcts/reform/civil/handler/callback/user/CreateSDOCallbackHandler.java

```diff
@@ -694,9 +688,112 @@ private String getBody(CaseData caseData) {
+    private void setCheckList(
+        CaseData.CaseDataBuilder<?, ?> updatedData,
+        List<OrderDetailsPagesSectionsToggle> checkList
+    ) {
+        updatedData.fastTrackAltDisputeResolutionToggle(checkList);
+        updatedData.fastTrackVariationOfDirectionsToggle(checkList);
+        updatedData.fastTrackSettlementToggle(checkList);
+        updatedData.fastTrackDisclosureOfDocumentsToggle(checkList);
+        updatedData.fastTrackWitnessOfFactToggle(checkList);
+        updatedData.fastTrackSchedulesOfLossToggle(checkList);
+        updatedData.fastTrackCostsToggle(checkList);
+        updatedData.fastTrackTrialToggle(checkList);
+        updatedData.fastTrackMethodToggle(checkList);
+        updatedData.disposalHearingDisclosureOfDocumentsToggle(checkList);
+        updatedData.disposalHearingWitnessOfFactToggle(checkList);
+        updatedData.disposalHearingMedicalEvidenceToggle(checkList);
+        updatedData.disposalHearingQuestionsToExpertsToggle(checkList);
+        updatedData.disposalHearingSchedulesOfLossToggle(checkList);
+        updatedData.disposalHearingFinalDisposalHearingToggle(checkList);
+        updatedData.disposalHearingMethodToggle(checkList);
+        updatedData.disposalHearingBundleToggle(checkList);
+        updatedData.disposalHearingClaimSettlingToggle(checkList);
+        updatedData.disposalHearingCostsToggle(checkList);
+        updatedData.disposalHearingApplicationsOrderToggle(checkList);
+        updatedData.smallClaimsHearingToggle(checkList);
+        updatedData.smallClaimsMethodToggle(checkList);
+        updatedData.smallClaimsDocumentsToggle(checkList);
+        updatedData.smallClaimsWitnessStatementToggle(checkList);
+        updatedData.disposalHearingApplicationsOrderToggle(checkList);
+    }
+
+    private List<String> setPreferredLocationFirst(List<LocationRefData> locations, CaseData caseData) {
+        String locationLabel;
+        if (caseData.getSuperClaimType() == SuperClaimType.SPEC_CLAIM) {
+            RequestedCourt courtRequest = Optional.ofNullable(caseData.getApplicant1DQ())
+                .map(Applicant1DQ::getRequestedCourt).orElse(null);
+            if (courtRequest != null && courtRequest.getRequestHearingAtSpecificCourt() == YesOrNo.YES) {
+                locationLabel = courtRequest.getResponseCourtCode();
+            } else {
+                return getLocationsFromList(locations);
+            }
+        } else {
+            // assume unspec
+            locationLabel = caseData.getCourtLocation().getApplicantPreferredCourt();
+        }
+
+        var preferredLocation =
+            locations
+                .stream()
+                .filter(locationRefData -> checkLocation(locationRefData,
+                                                         locationLabel)).findFirst();
+
+        if (caseData.getOrderType() == OrderType.DECIDE_DAMAGES && preferredLocation.isPresent()) {
```

Comment:
is there a reason we are not checking the following -
!(SPEC_CLAIM.equals(caseData.getSuperClaimType())
and using OrderType instead?